packages/orchestrator-ui-components/src/components/WfoForms/formFields/IpPrefixTable.tsx

```diff
@@ -0,0 +1,453 @@
+/*
+ * Copyright 2019-2023 SURF.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+import {
+    EuiBadge,
+    EuiButton,
+    EuiButtonIcon,
+    EuiCheckbox,
+    EuiFieldText,
+    EuiFlexGroup,
+    EuiFlexItem,
+    EuiFormRow,
+    EuiInMemoryTable,
+    EuiPanel,
+    EuiText,
+} from '@elastic/eui';
+import React from 'react';
+import ReactSelect, { SingleValue } from 'react-select';
+// import { getReactSelectTheme } from 'stylesheets/emotion/utils';
```

Comment:
commented import statement (not sure this is still on your todo)

Reply:
No.

This would involve: ClassBased -> Function based.

`const customStyles = getReactSelectTheme(this.context.theme);` -> but the context is already used for the ApiClient